### ISSUE TITLE
[api-extractor] Lost ability to link to getters and setters

### ISSUE THREAD

**Issue by acchou** [microsoft/rushstack#1616]
**Is this a feature or a bug?**

- [ ] Feature
- [x ] Bug

**Please describe the actual behavior.**

A recent change resulted in loss of the ability to link to getters and setters. Example:

```
/**
 * Comment about {@link FaastError.message}
 * @public
 */
export class FaastError extends VError {
    get message() {
        return super.message;
    }

    set message(value: string) {
        super.message = value;
    }
}
```

This worked in the past, e.g. see https://faastjs.org/docs/api/faastjs.faasterror. But with a recent (I think) change the following error emerges:

```
WARNING: Unable to resolve reference "FaastError.message": The member reference "message" was not found
Warning: dist/decls/src/error.d.ts:32:1 - (ae-unresolved-link) The @link reference could not be resolved: No member was found with name "message"
```

I'm attaching a screenshot showing the diff of the extracted `api.md` file:

<img width="1074" alt="Screen Shot 2019-11-06 at 11 03 52 AM" src="https://user-images.githubusercontent.com/1381213/68264854-377fe380-0085-11ea-97c8-bfe09a0e4b32.png">

**If this is a bug, please provide the tool version, Node.js version, and OS.**

* **Tool:** api-extractor 7.5.4, api-documenter 7.5.6
* **Tool Version:** See above
* **Node Version:** v12.13.0
  * **Is this a LTS version?** yes
  * **Have you tested on a LTS version?** yes
* **OS:** MacOS Catalina 10.15

**Comment by octogonz** [comment]
Could you provide a repro branch? That might help investigate faster.

**Comment by acchou** [comment]
The problem appears on the master branch of faast.js: https://github.com/faastjs/faast.js.
Not sure if you can see the circleci logs for it but it's here: https://app.circleci.com/jobs/github/faastjs/faast.js/3848

**Comment by acchou** [comment]
... or did you mean to create a repo with a small minimized example? :)

**Comment by octogonz** [comment]
I'm fine using the master branch if there are easy, clear instructions for building it

**Comment by acchou** [comment]
```
npm install
npm run build
```

The build step builds the documentation. To build the documentation only (requires a prior successful build as above), run:
```
npm run doc
```

**Comment by octogonz** [comment]
@acchou thanks for reporting this!  I started investigating it today.

In the API model, we need for the getter/setter pair to be merged into a single "Property" entry, but with two signatures (similar to the proposed [`@partOf`](https://github.com/microsoft/tsdoc/issues/137#issuecomment-446446564) TSDoc tag).  I don't think it would be useful to model them as separate API entities.  That seems like the main change required to get this working.

**Comment by octogonz** [comment]
@acchou PR https://github.com/microsoft/rushstack/pull/1646 should fix this issue. 

Could you test it out and confirm whether it solves your problem?

**Comment by acchou** [comment]
I’ll try it. What’s the easiest way to try it?

**Comment by octogonz** [comment]
If you want to run the CLI manually, you can build the rushstack repo [and then invoke lib/start.js](https://api-extractor.com/pages/contributing/debugging/).  Something like this:

```bash
$ cd my-project
$ node ../../../../rushstack/apps/api-extractor/lib/start.js markdown
```

Or, if your toolchain invokes API Extractor in a complicated way, you can just copy the lib folder into your node_modules folder.  Something like this:
```bash
$ cd my-project/node_modules/@microsoft/api-extractor
$ move lib lib.old
$ cp -Ra ../../../../rushstack/apps/api-extractor/lib ./lib
```

**Comment by acchou** [comment]
The PR appears to work as expected. Thanks!

**Comment by octogonz** [comment]
This fix was published with API Extractor 7.7.0
